FirebaseRemoteConfig/Tests/SwiftAPI/RemoteConfigConsole.swift

```diff
@@ -38,60 +38,53 @@ class RemoteConfigConsole {
     return accessToken
   }()
 
-  /// Synchronously fetches and returns currently active
-  /// remote config, if it exists.
+  /// Synchronously fetches and returns currently active Remote Config, if it exists.
   public var activeRemoteConfig: [String: Any]? {
     var config: [String: Any]?
     perform(configRequest: .get) { latestConfigJSON in
       config = latestConfigJSON
     }
     if let config = config {
-      save(config)
+      saveConfig(config)
     }
     return config
   }
 
-  /// Exposing this initializer allows us to
-  /// create`RemoteConfigConsole` instances without
-  /// dependence on a`GoogleService-Info.plist`.
+  /// Exposing this initializer allows us to create`RemoteConfigConsole` instances without
+  /// depending on a `GoogleService-Info.plist`.
   init(projectID: String) {
     self.projectID = projectID
     syncWithConsole()
   }
 
-  /// This initializer will attempt to read from a
-  /// `GoogleService-Info.plist` to set `projectID`.
+  /// This initializer will attempt to read from a `GoogleService-Info.plist` to set `projectID`.
   convenience init() {
     let currentBundle = Bundle(for: type(of: self))
-    let projectID = currentBundle.plistValue(for: "PROJECT_ID", from: "GoogleService-Info.plist")
+    let projectID = currentBundle.plistValue(
+      forKey: "PROJECT_ID",
+      fromPlist: "GoogleService-Info.plist"
+    )
     self.init(projectID: projectID! as! String)
   }
 
   // MARK: - Public API
 
-  public func updateRemoteConfig(parameters: [String: Any]) {
+  /// Update Remote Config with multiple String key value pairs.
+  /// - Parameter parameters: Dictionary representation of config key value pairs.
+  public func updateRemoteConfig(with parameters: [String: String]) {
     var updatedConfig: [String: Any] = latestConfig
 
     let latestParameters = latestConfig["parameters"] as? [String: Any]
-    if var updatedParameters = latestParameters {
-      for (key, value) in parameters {
-        updatedParameters.updateValue(["defaultValue": ["value": value]], forKey: key)
-      }
-      updatedConfig.updateValue(updatedParameters, forKey: "parameters")
-
-    } else {
-      var newParameters = [String: Any]()
-
-      for (key, value) in parameters {
-        newParameters.updateValue(["defaultValue": ["value": value]], forKey: key)
-      }
-      updatedConfig.updateValue(newParameters, forKey: "parameters")
+    var updatedParameters = latestParameters ?? [String: Any]()
+    for (key, value) in parameters {
+      updatedParameters.updateValue(["defaultValue": ["value": value]], forKey: key)
     }
+    updatedConfig.updateValue(updatedParameters, forKey: "parameters")
 
     publish(config: updatedConfig)
   }
 
-  public func updateRemoteConfigValue(_ value: Any, for key: String) {
+  public func updateRemoteConfigValue(_ value: String, forKey key: String) {
```

Comment:
Did you mean to change the `value` to be a `String` here? Seems like it should stay as `Any`, unless you found a good reason otherwise :)

Reply:
I realized that the value needs to be able to be convertible to a string. This [json object](https://firebase.google.com/docs/remote-config/automate-rc#rest-conditions) is pretty much what the RemoteConfigConsole API is sending to the servers.  Is there a way to check if an object can be used to construct a String like `let stringifiedThing = String(object)`?

Reply:
I guess the server logic can parse the config object out and infer the proper type for everything 😅